afdko/Tools/Programs/makeotf/makeotf_lib/source/cffread/cffread.c

```diff
@@ -86,7 +86,7 @@ typedef struct {                  /* FDArray element */
 typedef union {                   /* Stack element */
 	double d;
 	Fixed f;
-	long l;
+	int32_t l;
```

Comment:
The `l` was a `long`, but is now an `int32_t`, perhaps the name should be changed to `i` to reflect this?
(see related comments below)